### ISSUE TITLE
Nighly windows build and my own build fails to run ---darktable.exe caused an Access Violation at location 00007FF8A26965A0 in module libharfbuzz-0.dll ......

### ISSUE THREAD

**Issue by spyfish** [darktable-org/darktable#11407]
**Introduction**
It appears the windows builds are broken at the moment? 

At first I thought I did something wrong, as my builds failed on two different PCs. But now Ive also tested the nightly build at 
https://github.com/darktable-org/darktable/actions/workflows/nightly.yml 
 
Also fail with exact same error. 

Ive tried building 25/3 and 26/3 using this guide: 
https://github.com/darktable-org/darktable/blob/master/packaging/windows/BUILD.md 

A build based off 3.9.0+798 I grabbed via pixl.us works just fine.
 
Ive also reported at : 
https://discuss.pixls.us/t/problems-building-darktable-for-windows/30064

Latest i tested was darktable-3.9.0+820~g24e9d2015 (nightly build)
My own build today was 3.9.0+839~g97deab213 

---------------------------------------------------------------

**Describe the bug/issue**

When I try to run I get: 
![image](https://user-images.githubusercontent.com/5610400/160255575-299c5d53-af22-4d72-9af1-2744c3125ff9.png)

Back-trace:
[darktable_bt_9LSAJ1.txt](https://github.com/darktable-org/darktable/files/8356326/darktable_bt_9LSAJ1.txt)
[darktable_bt_JVKGJ1.txt](https://github.com/darktable-org/darktable/files/8356327/darktable_bt_JVKGJ1.txt)

**To Reproduce**
_Please provide detailed steps to reproduce the behaviour, for example:_
Install and try to run 

**Expected behavior**
Run

**Platform**

* darktable version : darktable-3.9.0+820~g24e9d2015 & 3.9.0+839~g97deab213  
* OS                : Win 10 21H2
* Memory            : 32GB
* Graphics card     : nvidia RTX 3080 
* Graphics driver   : NVIDIA Studio Driver | 511.09
* OpenCL installed  : yes
* OpenCL activated  : yes
* CMAKE_BUILD_TYPE  :  Ninja using MSYS2/mingw64

**Comment by kmilos** [comment]
This is a potential Msys2 issue, not a dt issue.

Let's check the dt nightly build again in a day or two when harfbuzz 4.1.0 appears in the Msys2 repo.

**Comment by kmilos** [comment]
Btw, thanks for catching this - exactly why we have the nightly build, so please keep testing them!

**Comment by kmilos** [comment]
harfbuzz-4.1.0-1 is available in the repo now (you’ll need to run pacman -Syu twice, with closing and reopening the MSYS terminal in between), so please try rebuilding now, or the nightly tomorrow (28/3)

**Comment by spyfish** [comment]
Thanks, I updated harfbuzz and rebuilt, but still same. Also the nightly from today fails the same.

**Comment by gi-man** [comment]
Do you have the logfile from darktable? Maybe with -d all?

**Comment by kmilos** [comment]
Would be good to confirm if an earlier harfbuzz works w/ darktable as expected.

harfbuzz packages before 4.x.x are available here: https://repo.msys2.org/mingw/mingw64/ and can be downgraded by pacman -U

**Comment by spyfish** [comment]
I do know Todd Prior made a build just 2 days ago that did work, so i very much suspect older harfbuzz will be OK. I am sorry i dont have the time to test this now myself.

**Comment by kmilos** [comment]
So downgrading to harfbuzz-3.4.0 doesn't help - got a different crash due to pango, so I'm trying w/ matching downgraded pango version as well...

**Comment by kmilos** [comment]
Yep, downgrading to both `harfbuzz-3.4.0-1` and `pango-1.50.5-1` did the trick.

**Comment by spyfish** [comment]
> Yep, downgrading to both `harfbuzz-3.4.0-1` and `pango-1.50.5-1` did the trick.

Indeed it did, thanks allot :)

**Comment by kmilos** [comment]
Btw, manually adding explicit fonts in `share/darktable/themes/darktable.css` also works around the issue w/ the OOTB nightly builds for the moment. Change all `font-family` occurrences like so for example:

```
font-family: "Segoe UI", sans-serif;
font-family: "Segoe UI", sans;
font-family: Consolas, monospace;
```

**Comment by kmilos** [comment]
A [patched pango-1.50.6-3 is now available in the MSYS2 repo](https://github.com/msys2/MINGW-packages/commit/99232723b6d7b10517ca15eacdc9d10cff441341), and the next nightly should be working again.
